development.txt

```diff
@@ -133,6 +133,12 @@ Some more packages are recommended but not mandatory for a working LibreMesh net
 - LiMe -> Offline Documentation -> lime-docs (LibreMesh English documentation)
 - LiMe -> lime-hwd-ground-routing (Manage 802.1q VLANs for ground routing)
 - LiMe -> lime-debug (libremesh debug utils)
+- Network -> wpad-mesh-wolfssl (for password protected 802.11s mesh)
+
+Additionally and optionally, httpS for the web interface can be enabled selecting (beware that the web interace will be shown as *not trusted*):
+
+- Libraries -> libustream-wolfssl
+- Utilities -> Encryption -> px5g-standalone
```

Comment:
Wow thanks for maintaining that package @aparcar !
But the idea is to have the LibreMesh 20.09 possible to compile on OpenWrt 18.06 and OpenWrt 19.07 and [px5g-wolfssl](https://github.com/openwrt/openwrt/tree/master/package/utils/px5g-wolfssl) is currently present only in snapshot.
Do you know if it will be pulled also in OpenWrt 18.06 and 19.07?
Thanks!

Reply:
Oh I don't think so. I though we're updating the docs for the next release

Reply:
Well, let's try to get one release done first :) 
Once we have that, I think that we should re-organize the "get it" and the "development" pages to track respectively the release and the future release.